api/user.go

```diff
@@ -1007,8 +1007,11 @@ func getProfiles(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 
 	etag := (<-Srv.Store.User().GetEtagForProfiles(id)).Data.(string)
-	if HandleEtag(etag, w, r) {
-		return
+	if !HasPermissionToContext(c, model.PERMISSION_MANAGE_SYSTEM) {
```

Comment:
sure the thing here is that if the user is a `System Admin` who is doing the request we bypass the etag and that way it will always return the profiles with the `AuthService`.

now that I can think of it, the etag will have `AuthService` if the user is a system_admin, I'll update the PR